pandas/core/series.py

```diff
@@ -1002,24 +1002,33 @@ def _set_value(self, label, value, takeable=False):
 
     def reset_index(self, level=None, drop=False, name=None, inplace=False):
         """
-        Analogous to the :meth:`pandas.DataFrame.reset_index` function, see
-        docstring there.
+        Reset the index of the Serie.
```

Comment:
General comment: Serie -> Series (the singular of Series is still Series :))

Reply:
Solved tnx!

Reply:
Did you already push? Because this one is not yet fixed

Reply:
Sorry, I missed it!